trlx/data/configs.py

```diff
@@ -62,6 +66,9 @@ class TrainConfig:
 
     :param batch_size: Batch size for training
     :type batch_size: int
+    
+    :param accumulate_steps: How many forward calls before the optimizer takes a step
+    :type accumulate_steps: int
```

Comment:
Re-name this to `gradient_accumulation_steps` to match other APIs ([DeepSpeed](https://www.deepspeed.ai/docs/config-json) and [Accelerate](https://huggingface.co/docs/accelerate/v0.15.0/en/package_reference/cli#accelerate-launch)).